setup.py

```diff
@@ -7,7 +7,7 @@ def readme():
 
 setup(
     name='drf-jsonapi',
-    version='0.1.0',
+    version='0.1.3',
```

Comment:
This is set via `setuptools_scm` now, hence the conflict with `setup.py` that GitHub is mentioning. Need to decide if we want to continue with that or use a different workflow. @kelly-vacasa and I will discuss today.